files/setup

```diff
@@ -63,4 +74,14 @@ You should then be able to autoindent with Cmd-Shift-B (Mac) or Ctrl-Shift-B
 (Windows).
 HEREDOC
 
+  if runs > 0
+    puts <<-SECOND_RUN
```

Comment:
Hi! It looks like you've run `bin/setup` more than once for this project. That's unusual; are you sure that's what you want to do?

 - Maybe you're trying to reset the dummy data in your development database;
 if so, try

    rails dev:prime

    instead.

 - Maybe you're trying to enable autoformatting for .rb and .html.erb files. If so,

    1. In the Cloud9 menu, select Open Your Init Script and add the following code:

        services.pluginManager.loadPackage([
          "~/.c9/plugins/formatruby/package.json",
          "~/.c9/plugins/formathtmlerb/package.json",
        ]);

    (if it's not already there).

    2. In the Cloud9 menu, select Preferences > Plugin Explorer and check "formathtmlerb" and "formatruby".